### PR TITLE
[DOC] Update CUDA requirements in documentation and Dockerfiles[skip ci]

### PR DESCRIPTION
**Pull request by nvliyuan**
This pr is to address document issues:
1.  #8641
2.  #8653
3. update out-of-date getting started gcp doc
Signed-off-by: liyuan <yuali@nvidia.com>

### CODE REVIEW

**Comment by nvliyuan** [comment]
For #8641, updated the CUDA container images versions that will be deleted from NGC and Docker Hub:
```
(base) yuXXXX:XXXXX XXXX$ grep -rn "cuda:" *
docs/additional-functionality/shuffle-docker-examples/Dockerfile.rocky_no_rdma:29:FROM nvidia/cuda:${CUDA_VER}-runtime-rockylinux${ROCKY_VER}
docs/additional-functionality/shuffle-docker-examples/Dockerfile.ubuntu_rdma:61:FROM nvidia/cuda:${CUDA_VER}-runtime-ubuntu${UBUNTU_VER}
docs/additional-functionality/shuffle-docker-examples/Dockerfile.rocky_rdma:29:FROM nvidia/cuda:${CUDA_VER}-runtime-rockylinux${ROCKY_VER}
docs/additional-functionality/shuffle-docker-examples/Dockerfile.ubuntu_no_rdma:31:FROM nvidia/cuda:${CUDA_VER}-runtime-ubuntu${UBUNTU_VER}
docs/get-started/Dockerfile.cuda:17:FROM nvidia/cuda:11.8.0-devel-ubuntu18.04
jenkins/Dockerfile-blossom.ubuntu:32:FROM nvidia/cuda:${CUDA_VER}-runtime-ubuntu${UBUNTU_VER}
jenkins/Dockerfile-blossom.integration.centos:28:FROM nvidia/cuda:${CUDA_VER}-runtime-centos${CENTOS_VER}
jenkins/Dockerfile-blossom.integration.ubuntu:29:FROM nvidia/cuda:${CUDA_VER}-runtime-ubuntu${UBUNTU_VER}
jenkins/Dockerfile-blossom.integration.rocky:28:FROM nvidia/cuda:${CUDA_VER}-runtime-rockylinux${ROCKY_VER}

```

For #8653, update v450.80.02+  to v470+ to keep sync with the [cuda document](https://docs.nvidia.com/datacenter/tesla/drivers/#cuda-drivers)

**Review by jlowe — COMMENTED** [review]
This is modifying more than just documentation, it's also modifying Dockerfiles used during CI.  Do we really want to skip premerge CI for this, or are these Dockerfile changes not going to be tested by a premerge CI run?

**Comment by sameerz** [comment]
> This is modifying more than just documentation, it's also modifying Dockerfiles used during CI. Do we really want to skip premerge CI for this, or are these Dockerfile changes not going to be tested by a premerge CI run?

@NvTimLiu can you confirm whether the changes to the jenkins dockerfiles will be tested in pre-merge?

**Review by viadea — DISMISSED** [review]
Approved the content(before confirming if we need to skip CI)

**Comment by nvliyuan** [comment]
removed the skip-ci tag

File: jenkins/Dockerfile-blossom.ubuntu [hunk 1]
```diff
@@ -25,7 +25,7 @@
 #       UCX_VER=1.13.1
 ###
 
-ARG CUDA_VER=11.0.3
```
Comment (pxLi) [inline jenkins/Dockerfile-blossom.ubuntu:28]: BTW https://hub.docker.com/r/nvidia/cuda/tags?page=1&name=11.0.3 is still supported, I guess we may not update this one as we still trying to use the minimal cuda version image in our nightly and premerge.

File: jenkins/Dockerfile-blossom.integration.centos [hunk 2]
```diff
@@ -23,7 +23,7 @@
 #    URM_URL=<maven repo url>
 ###
 
-ARG CUDA_VER=11.0
```
Comment (pxLi) [inline jenkins/Dockerfile-blossom.integration.centos:26]: same for 11.0

File: jenkins/Dockerfile-blossom.ubuntu (hunk 1 above)
Comment (nvliyuan) [reply jenkins/Dockerfile-blossom.ubuntu]: revert the change, thanks

File: jenkins/Dockerfile-blossom.integration.centos (hunk 2 above)
Comment (nvliyuan) [reply jenkins/Dockerfile-blossom.integration.centos]: revert the change, thanks

**Review by pxLi — APPROVED** [review]
thanks for taking care of this. As no code change for premerge, we could [skip ci] now

**Review by NvTimLiu — APPROVED** [review]
LGTM, I'll update github/examples related dockerfiles/images. thx~

**Review by sameerz — APPROVED** [review]

**Comment by nvliyuan** [comment]
build

**Pull request merged**